### DISCUSSION TITLE
Launch kubernetes-dashboard in a logged-in state

### DISCUSSION THREAD

**Discussion by vanyans** [kubernetes/dashboard#9650] [Q&A]
Hello.
I have a web app that provides links to all system's dashboards for user's convenience.
It has a button for kubernetes-dashboard too.
When a user clicks on a button, i would like to open a new page with Kubernetes dashboard and i would like to automatically authenticate user. ServiceAccount token would be generated every time.

When I test this with Requestly it works. When I add generated token to haproxy it works. But I want the token to be dynamic.
I tried using the following format https://k8s-dashboard/dashboard/?token=<token>
dashboard didn't like it. haproxy had issues extracting the token with urlp(token). couldn't get it to work.
I tried to pass request header in my app's backend but that didn't work either. 

Any suggestions would be appreciated.

Thanks

**Comment by joseph-skupniewicz-sp** [comment]
Hey @vanyans did you ever come up with a solution for this? Maybe some of the info in discussion https://github.com/kubernetes/dashboard/discussions/8980 might help or provide further ideas?
